zcash_client_sqlite/src/transact.rs

```diff
@@ -120,8 +123,9 @@ struct SelectedNoteRow {
 ///     Err(e) => (),
 /// }
 /// ```
-pub fn create_to_address<P: AsRef<Path>>(
-    db_data: P,
+pub fn create_to_address<DB: AsRef<Path>, P: consensus::Parameters + Clone>(
```

Comment:
I think I'd prefer that we add a `Clone` bound to the `consensus::Parameters` trait itself (_all_ network parameters should be cloneable).